custom_search_commands/python/customsearchcommands_app/metadata/default.meta

```diff
@@ -1,2 +1,3 @@
 []
 access = read: [ * ], write : [ admin ]
+export = system
```

Comment:
Is this related to resolving a particular appinspect issue? If so can you give some detail?

Reply:
not related to appinspect issue, this change was done so that we can run custom commands from other(search) app.